crates/wasi-common/cap-std-sync/Cargo.toml

```diff
@@ -13,13 +13,14 @@ include = ["src/**/*", "LICENSE" ]
 
 [dependencies]
 wasi-common = { path = "../", version = "0.26.0" }
+wiggle = { path = "../../wiggle", version = "0.26.0" }
```

Comment:
I think this is only used for the `async_trait` macro, right? If that's the case could this be just the `async_trait` dependency?

Reply:
Yep thats the only use. I arbitrarily decided to import it via wiggle everywhere so we wouldn't have to keep versions synced up, but realistically I don't expect async_trait to change meaningfully at this point so I could change to importing it directly.